packages/library-base/src/commonMain/kotlin/io/realm/schema/RealmSchema.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.schema
+
+interface RealmSchema {
+    val classes: Collection<RealmClass>
```

Comment:
Should we also expose the schema version here? Or does that causes problems during migrations 🤔

Reply:
I also iterated the thought a couple of times, but didn't find the magic bullet. I think we should expose it on an immutable schema, but since I haven't landed the _schema refresh_ parts of the key-based accessors, I am in doubt on the exact requirements and what would make sense. 

Currently the compiler plugin builds the embedded schema metadata directly with the cinterop-types and haven't change to use the public API as container during compile time, but if we change to use the public API types through out compiler plugin too, then we would have to be able to represent a schema without a _transaction version_ (input is welcome). 

Will add a todo and see if I can settle it as part of #600

Reply:
Yes, we can always come back to it once other parts settle. Just adding a TODO is 👍